_styles/-theme.scss

```diff
@@ -3,23 +3,25 @@
 
 // colors
 [data-dark="false"] {
-  --primary: #0ea5e9;
+  --primary: #0795d9;
```

Comment:
Double checking: are the primary color of `data-dark="true"` and `data-dark="false"` supposed to be the same?

Reply:
Yes, same. It's roughly in the middle of brightness and has adequate contrast (ignoring the sometimes questionable WCAG color constrast metrics we discussed previously) with dark and light bgs.